utilities/blob_db/blob_db_impl.cc

```diff
@@ -1703,15 +1708,31 @@ std::pair<bool, int64_t> BlobDBImpl::SanityCheck(bool aborted) {
       pos += snprintf(buf + pos, sizeof(buf) - pos,
                       ", expiration range (%" PRIu64 ", %" PRIu64 ")",
                       expiration_range.first, expiration_range.second);
+      if (pos >= static_cast<int>(sizeof(buf))) {
```

Comment:
If the check on line 1695 is true, so must the check on line 1708.  Can we just log the same error message once?